pkg/imageverifycache/client.go

```diff
@@ -25,7 +25,9 @@ type cache struct {
 type Option = func(*cache) error
 
 func New(options ...Option) (Client, error) {
-	cache := &cache{}
+	cache := &cache{
+		isCacheEnabled: true,
```

Comment:
Is there a way to custom the default cache to false?

Reply:
Yes, there is an option to disable the cache: https://github.com/kyverno/kyverno/blob/6ad156f5d0eae61022d7635b1254111fe99542bd/pkg/imageverifycache/client.go#L63-L68

Reply:
Why do we want to initialize the cache with `isCacheEnabled: true`? The default flag `imageVerifyCacheEnabled` is set to true, can we use that to set up the cache?

Reply:
yes, and this wont change that behaviour, but if someone uses this struct then they would assume that it is enabled by default.

Reply:
I would add a new function to initialize the client with cache enabled, the init function should be non-biased IMHO.

Reply:
makes sense, I removed it, since we don't need one at the moment and we can enable it in the init function, I am not adding the new function